tasks/configure_gitlab_ce.yml

```diff
@@ -155,20 +155,46 @@
     mode: '0644'
   notify: [ 'Restart gitlab' ]
 
-- name: Install GitLab service script
+- name: Install GitLab SysVinit script
   shell: cp -f {{ gitlab_ce_git_checkout + "/lib/support/init.d/gitlab" }} /etc/init.d/gitlab
   register: gitlab__register_init_script
-  when: gitlab_register_ce_checkout is defined and gitlab_register_ce_checkout.changed
+  when: ((gitlab_register_ce_checkout is defined and gitlab_register_ce_checkout.changed) and
+         (gitlab_use_systemd is defined and not gitlab_use_systemd))
+
+- name: Install GitLab systemd service files
+  template:
+    src: 'etc/systemd/system/{{ item }}.j2'
+    dest: '/etc/systemd/system/{{ item }}'
+  with_items:
+    - 'gitlab-mailroom.service'
+    - 'gitlab-sidekiq.service'
+    - 'gitlab-unicorn.service'
+    - 'gitlab-workhorse.service'
+  register: gitlab__register_init_script
+  when: (( gitlab_register_ce_checkout is defined and gitlab_register_ce_checkout.changed) and
+         (gitlab_use_systemd is defined and gitlab_use_systemd))
 
 - name: Reload systemd daemons
   command: systemctl daemon-reload
   when: (ansible_service_mgr == "systemd" and gitlab__register_init_script|changed)
 
-- name: Enable GitLab service
+- name: Enable GitLab SysVinit service
   service:
     name: 'gitlab'
     enabled: True
-  when: gitlab_register_ce_checkout is defined and gitlab_register_ce_checkout.changed
+  when: ((gitlab_register_ce_checkout is defined and gitlab_register_ce_checkout.changed) and
+         (gitlab_use_systemd is defined and not gitlab_use_systemd))
+
+- name: Enable GitLab systemd services
+  service:
+    name: '{{ item }}'
+    enabled: True
+  with_items:
+    - 'gitlab-sidekiq.service'
+    - 'gitlab-unicorn.service'
+    - 'gitlab-workhorse.service'
```

Comment:
The `gitlab-mailroom.service` is not enabled or started anywhere after it's installed. Is that by design? When I started it manually, the process shows up in `systemctl status` output:
```
           │ ├─gitlab-mailroom.service
           │ │ └─29355 ruby2.1 /var/local/git/gitlab/vendor/bundle/ruby/2.1.0/bin/mail_room -q -c /var/local/git/gitlab/config/mail_room.yml
```
But the service itself doesn't seem to be able to show it like other services:
```
● gitlab-mailroom.service - Gitlab mailroom Worker
   Loaded: loaded (/etc/systemd/system/gitlab-mailroom.service; enabled)
   Active: active (exited) since pon 2017-02-13 13:07:23 CET; 4min 27s ago
  Process: 29347 ExecStart=/var/local/git/gitlab/bin/mail_room start (code=exited, status=0/SUCCESS)
 Main PID: 29347 (code=exited, status=0/SUCCESS)

lut 13 13:07:23 code systemd[1]: Starting Gitlab mailroom Worker...
lut 13 13:07:23 code systemd[1]: Started Gitlab mailroom Worker.
```
Maybe this depends on the unit type used and could be fixed?